docker-compose.yml

```diff
@@ -1,31 +1,12 @@
 version: "3.8"
 
 services:
-  mysql:
+  db:
     image: mysql:8.0.23
     environment:
-      MYSQL_DATABASE: 'yourdbname'
-      MYSQL_USER: 'user'
-      MYSQL_PASSWORD: 'password'
-      MYSQL_ROOT_PASSWORD: 'rootpassword'
+      MYSQL_DATABASE: puresynth
+      MYSQL_USERNAME: ${DB_USERNAME}
```

Comment:
It's ok to hardcode credentials here, because it's just for testing purposes and everyone need to create their own instance.